### DISCUSSION TITLE
Keyboard shortcut to edit an Issue's comment

### DISCUSSION THREAD

**Discussion by lance-anderson** [community/community#4279] [Projects and Issues]
I am frequently editing the main comment of an Issue adding inline updates as an Issue/Epic/User Story evolves. It would be super rad to have a keyboard shortcut (perhaps `command+e`) to start editing a comment.

I didn't see anything like it here:
https://docs.github.com/en/get-started/using-github/keyboard-shortcuts

Although I did learn that `command+shift+p` will toggle between the Write and Preview tabs which is nice. And now that I think about it, perhaps using `command+shift+p` to open the Write tab may even be better than `command+e` or some other key command.

To be clear, I'm asking for a keyboard shortcut to execute this Edit option:
![image](https://user-images.githubusercontent.com/3789349/123203772-b5b41800-d46b-11eb-9a4e-99ee71170615.png)
Thank you!!

**Comment by mikelyncheski** [comment]
Yes!  

I find it odd that there is a nice Edit Button on top that gives one click access to editing the title which I rarely do and that it takes a click, a mouse move, and another click to change the comment which I do all day long.  We keep checklists of to do items on our larger issues and the list is always evolving.

Another option would be to just click on the comment text to enable the edit mode.

Thanks!

**Reply by weitzhandler** [reply]
Or double click to avoid accidental edits.

**Comment by nexushoratio** [comment]
I was looking for this as well.

While not a single key, I did just learn that the command palette (default, brought up by `ctrl-k`) has `Edit issue title` as the first command when viewing an issue.   So, pretty quick.

**Reply by treeder** [reply]
Great tip, thanks for sharing!

**Reply by weitzhandler** [reply]
That was very helpful, it also contains an Edit Body command which is really useful!

**Comment by VladimirAlexiev** [comment]
After I post a complex comment, I often edit it a couple of times until it's most informative and looks perfect.
In Slack, the `Up` arrow edits the **previous (last)** comment. Can Github offer the same function?

**Comment by midenok** [comment]
Any progress on this? There was the same nice button without submenu to edit the comment. Any problem to return it back?
